[2025-09-21T11:08:03.039+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:07:00+00:00/task_id=fetch_training_data permission to 509
[2025-09-21T11:08:03.041+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:07:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:07:00+00:00/task_id=fetch_training_data'
[2025-09-21T11:08:03.095+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:07:00+00:00/task_id=fetch_training_data permission to 509
[2025-09-21T11:08:03.096+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:07:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:07:00+00:00/task_id=fetch_training_data'
[2025-09-21T11:08:03.150+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-09-21T11:07:00+00:00 [queued]>
[2025-09-21T11:08:03.163+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-09-21T11:07:00+00:00 [queued]>
[2025-09-21T11:08:03.163+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-09-21T11:08:03.189+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): fetch_training_data> on 2025-09-21 11:07:00+00:00
[2025-09-21T11:08:03.208+0000] {standard_task_runner.py:57} INFO - Started process 286 to run task
[2025-09-21T11:08:03.214+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'fetch_training_data', 'scheduled__2025-09-21T11:07:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmpof02x2ko']
[2025-09-21T11:08:03.218+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask fetch_training_data
[2025-09-21T11:08:03.302+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:07:00+00:00/task_id=fetch_training_data permission to 509
[2025-09-21T11:08:03.303+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:07:00+00:00/task_id=fetch_training_data permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:07:00+00:00/task_id=fetch_training_data'
[2025-09-21T11:08:03.307+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.fetch_training_data scheduled__2025-09-21T11:07:00+00:00 [running]> on host 36da5371e73d
[2025-09-21T11:08:03.446+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='fetch_training_data' AIRFLOW_CTX_EXECUTION_DATE='2025-09-21T11:07:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-21T11:07:00+00:00'
[2025-09-21T11:08:03.461+0000] {db_utils.py:17} INFO - Connected to PostgreSQL
[2025-09-21T11:08:04.709+0000] {train_predict_to_db_dag.py:29} INFO - Fetched training data tail:
            id  store  ...  productname                 created_at
100490  155930    431  ...    product_A 2025-09-21 11:07:16.545768
100491  155931    432  ...    product_A 2025-09-21 11:07:26.765075
100492  155932    426  ...    product_A 2025-09-21 11:07:36.734997
100493  155933    486  ...    product_A 2025-09-21 11:07:48.583266
100494  155934    451  ...    product_A 2025-09-21 11:07:59.514419

[5 rows x 12 columns]
[2025-09-21T11:08:04.803+0000] {python.py:194} INFO - Done. Returned value was: {'store': 370, 'start_date': '2025-06-23', 'end_date': '2025-09-21'}
[2025-09-21T11:08:05.069+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=fetch_training_data, execution_date=20250921T110700, start_date=20250921T110803, end_date=20250921T110805
[2025-09-21T11:08:05.166+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-21T11:08:05.213+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
